submissions/peterIyiola/html/productPage-1.html

```diff
@@ -0,0 +1,94 @@
+
+
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <title>Better Buys</title>
+   </head>
+   <body>
+      <nav>
+         <a href="#">
+            <h1><b>Better Buys</b></h1>
+         </a>
+         <h4><i>Your neighbourhood electronics store</i></h4>
+         <hr>
+         <div>
+            <input type="text" name="box" placeholder="Type your Search here"</input>
+            <button>Search</button> <br>
+            <br>
+            <button>Signin</button>
+            <button>Register</button>
+         </div>
+         <!--What we sell-->
+         <div>
+            <h3>Check us out for your:</h3>
+            <ul>
+               <li>TVs</li>
+               <li>Sound Systems</li>
+               </Sound>
+               <li>Air Conditioners</li>
+               <li>Refrigertors</li>
+            </ul>
+         </div>
+         <section>
+            <!--Product Images and Description-->
+            <div>
+               <ul>
+                  <li>
+                     <a href="#"><img src="TVs.jpg" alt="Smart TV"></a>
+                     <p>65 inches QLED SMART TV 
+                        <br>
+                        +Wall Brackets and 12 Months warranty<br>
+                        Price = 250,000
+                     </p>
+                  </li>
+                  <li>
+                     <a href="#"><img src="HT.jpg" alt="Home Theatre" ></a></
+                     <a>
+                        <p>Samsung Home Theatre System <br>
+                           Price=120,000 <br>
+                           High Definition Surround System
+                           Bluetooth Enabled
+                        </p>
+                  </li>
+                  <hr>
+                  <p>Thermocool 2HP Air conditioner <br>
+                  Price=170,000</p>
+                  <hr>  
+                  <p>HisenseRegrigerators<br>
+                  Price =200,000 
+                  <br>
+                  Hisense 24Litres Refrigerator<br>
+                  Buttom Freezer and Water Dispenser
+                  </p>
+               </ul>
+               <nav>
```

Comment:
Why do you have more than one `nav` tag. You're are suppose to have only one in an html file.